### ISSUE TITLE
Prone X Offset slider

### ISSUE THREAD

**Issue by wyretm** [tr7zw/FirstPersonModel#451]
Would be nice to have since should the player size be larger than usual, via Pehkui or anything else, this becomes your view while prone
![2024-05-20_14 59 54](https://github.com/tr7zw/FirstPersonModel/assets/170327040/3a334592-90b0-4fd2-bd9d-d16c801ce308)

Otherwise lovely mod, can't play without it.

**Comment by wyretm** [comment]
Thanks for the quick response, I understand that this mod doesn't move the camera, what I was suggesting is the ability to adjust X Offset for the prone position, the same way you can for standing and sitting.

**Comment by tr7zw** [comment]
The x offset is adjusted and has a slider. Pehkui breaks other mods so they don't know the player is actually sneaking, so Pehkui can do their own offset logic, which in return breaks this mod.

**Comment by wyretm** [comment]
To clarify as it seems there's a bit of a misunderstanding happening, I'm not talking about sneaking, I'm referring to being prone, as in lying on the ground. In any case I'm not going to bother you further, thank you for your work.

**Comment by tr7zw** [comment]
Ah, so you mean the "swimming on land"(literally what it is in terms of what Minecraft thinks, there is no "prone" code/logic). That might actually be something to look into. Then I assume swimming is affected the same way(at least when swimming straight, not up/down)?

**Comment by wyretm** [comment]
Yes, sorry for the confusion, and yes normal swimming is also affected.
